CHANGES.rst

```diff
@@ -12,6 +12,10 @@ assign_wcs
 - Changed evaluation of grism bounding box center from averaged extrema of
   transformed bounding box to transformed centroid of source_cat object [#5809]
 
+- Added pixel shift to non-fixed slits due to 0-indexing in NIRSpec slit
```

Comment:
Maybe phrase this in the positive sense, rather than negative, i.e. "Added pixel shift to MSA slits due to ..."

Reply:
Covered this in the most recent commit.